packages/react-components/react-spinbutton/src/stories/SpinButton/SpinButtonAccessibility.stories.mdx

```diff
@@ -93,26 +103,20 @@ How this is read also depends on the screen reader used but a screen reader shou
 
 ### Styling
```

Comment:
is this just a stub for later? should we ommit empty areas for now.

Reply:
Yup, though that wasn’t added in this PR. For now I’m just moving the files and updating the docs nav structure, and sometime this week I plan to fill in these sections 😄